engineering-decision-records/008-logbook.md

```diff
@@ -0,0 +1,33 @@
+---
+type: process
+status: accepted
+---
+
+# Logbooks
+
+## Context
+We introduce the notion of a "Logbook" to complement the [Engineering Decision Records](./001-record-engineering-decisions.md) process by capturing developer musings, notes, tips, and informal documentation throughout the development process.
+
+## Motivation
+[Engineering Decision Records](./001-record-engineering-decisions.md) are used within this repository to record official decisions that we reach as a team. However, they do not capture all context that might be useful to capture for a highly exploratory project like this one.
+For example, it would not be appropriate to record experiments, helpful tips, meeting notes, or other informal but highly valuable contextual information about development.
+We take inspiration from the [Ouroboros Leios](https://github.com/input-output-hk/ouroboros-leios/blob/main/Logbook.md) and [Ouroboros Peras](https://github.com/input-output-hk/peras-design/blob/main/Logbook.md) projects who instituted a "Logbook" to great effect.
+Finally, we want to avoid overly onerous process obsession, so use of the logbook will be encouraged but not mandatory or policed.
+
+## Decision
+
+- We will create a "Logbook" directory intended to capture these kinds of informal notes.
+- It will have a `team.md` file with a series of entries headings with the date in descending order.
```

Comment:
```suggestion
- It will have a `log.md` file with a series of entries headings with the date in descending order.
```

This seems inconsistent with the `log.md` created in the directory

Reply:
I figured `Logbook/log.md` was redundant, if we're going to have a directory with a mixture of other markdown files; The idea behind the name `team` specifically was if anyone ever felt compelled to make their own personal logbook (seems overkill right now, but who knows down the road), this one would serve as the shared team file.

Reply:
that being said I have no strong opinion here 🤷

Reply:
If it's overkill, then perhaps let's not do it? Plus the whole purpose of a logbook is to be shared, so why would anyone want to make their own? I don't have a strong opinion on the name of the file either.